lib/404.php

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
We can use the same HTTP header as on the `html.php` template.

```suggestion
<?php
header('Content-type: text/html; charset="UTF-8"');
?>
<!DOCTYPE html>
```